inc/projecttasktemplate.class.php

```diff
@@ -0,0 +1,260 @@
+         'id'       => '8',
+         'name'     => __('Milestone'),
+         'field'    => 'is_milestone',
+         'table'    => $this->getTable(),
+         'datatype' => 'bool',
+      ];
+
+      $tab[] = [
+         'id'       => '9',
+         'name'     => __('Planned start date'),
+         'field'    => 'plan_start_date',
+         'table'    => $this->getTable(),
+         'datatype' => 'datetime',
+      ];
+
+      $tab[] = [
+         'id'       => '10',
+         'name'     => __('Real start date'),
+         'field'    => 'real_start_date',
+         'table'    => $this->getTable(),
+         'datatype' => 'datetime',
+      ];
+
+      $tab[] = [
+         'id'       => '11',
+         'name'     => __('Planned end date'),
+         'field'    => 'plan_end_date',
+         'table'    => $this->getTable(),
+         'datatype' => 'datetime',
+      ];
+
+      $tab[] = [
+         'id'       => '12',
+         'name'     => __('Real end date'),
+         'field'    => 'real_end_date',
+         'table'    => $this->getTable(),
+         'datatype' => 'datetime',
+      ];
+
+      $tab[] = [
+         'id'       => '13',
+         'name'     => __('Planned duration'),
+         'field'    => 'planned_duration',
+         'table'    => $this->getTable(),
+         'datatype' => 'actiontime',
+      ];
+
+      $tab[] = [
+         'id'       => '14',
+         'name'     => __('Effective duration'),
+         'field'    => 'effective_duration',
+         'table'    => $this->getTable(),
+         'datatype' => 'actiontime',
+      ];
+
+      $tab[] = [
+         'id'       => '15',
+         'name'     => __('Description'),
+         'field'    => 'description',
+         'table'    => $this->getTable(),
+         'datatype' => 'textarea',
+      ];
+
+      return $tab;
+   }
+
+
+   /**
+    * @see CommonDropdown::displaySpecificTypeField()
+   **/
+   function displaySpecificTypeField($ID, $field = []) {
+
+      switch ($field['type']) {
+         case 'percent_done' :
+            Dropdown::showNumber("percent_done", ['value' => $this->fields['percent_done'],
+                                                  'min'   => 0,
+                                                  'max'   => 100,
+                                                  'step'  => 5,
+                                                  'unit'  => '%']);
+            break;
+         case 'actiontime' :
+            Dropdown::showTimeStamp($field["name"],
+                                    ['min'             => 0,
+                                     'max'             => 100 * HOUR_TIMESTAMP,
+                                     'step'            => HOUR_TIMESTAMP,
+                                     'value'           => $this->fields[$field["name"]],
+                                     'addfirstminutes' => true,
+                                     'inhours'         => true]);
+            break;
+      }
+   }
+
+
+   /**
+    * @param $field
```

Comment:
Incorrect doc. Drop entire phpdoc block since this method is inherited without changes.